lit.ts

```diff
@@ -99,6 +99,52 @@ export async function storeConditionWithSigner(
 	return tx;
 }
 
+export async function mintPKPV2({
+	keyType,
+	permittedAuthMethodTypes,
+	permittedAuthMethodIds,
+	permittedAuthMethodPubkeys,
+	permittedAuthMethodScopes,
+	addPkpEthAddressAsPermittedAddress,
+	sendPkpToItself,
```

Comment:
is there a reason to type inline like this if its already defined as an interface?

Reply:
how else do you mean this can be done?